README.md

```diff
@@ -233,7 +233,7 @@ So the above output can be read as "I assert, Person's name (which is actually `
 :information_source: You can also use `it` as alternative for `subject`. The above could have been written as:
 ```kotlin
 assert(person) {
-    property(it::name).toBe("Peter")
+    property(it::name).toBe("Peter") // The assertion subject is also passed as an implicit lambda parameter so you can use it
```

Comment:
That's actually not true. `it` is a property as well. I added it because people are used to use `it` as shortcut for whatever their main focus is.

Reply:
Why not use lambda parameter instead? This creates confusion...

Reply:
Yeah I see that it creates confusion. I should either remove `it` entirely or pass it as param. I am not sure which is better. Right now, I think passing it as param is overkill. It is a lambda with receiver where `subject` is available anyway. What do you think?

Reply:
If only `subject` is needed, then I'd not add any receiver, and I'd just pass the subject as single lambda parameter, giving `it` and the freedom to name it as one likes.
If the receiver is needed, then I think keeping `subject` only would be right and avoid any confusion or ambiguity.

Reply:
It has to be a receiver lambda, otherwise you could not use the assertion functions (they are all defined on IAssertionPlant). I'll consider to remove `it`. Thank you very much for your time and for your feedback :-)

Reply:
You're welcome :-)